primitives/src/lib.rs

```diff
@@ -277,6 +300,14 @@ pub mod redeem {
         pub btc_height: u32,
         /// the status of this redeem request
         pub status: RedeemRequestStatus,
+        /// the issue id
+        pub issue_id: Option<H256>,
+    }
+
+    impl<AccountId, BlockNumber, Balance, CurrencyId: Copy> RedeemRequest<AccountId, BlockNumber, Balance, CurrencyId> {
```

Comment:
need to explicitly mention the trait otherwise it will pass out an error `Copy` not implemented, check implementation for `VaultId` [here](https://github.com/interlay/interbtc/blob/878b15d9bc9deb6f3161e360582a99bc0ad5f430/primitives/src/lib.rs#L94)